sympy/integrals/tests/test_transforms.py

```diff
@@ -574,36 +574,48 @@ def test_issue_8368_7173():
 
 
 def test_inverse_laplace_transform():
-    from sympy import sinh, cosh, besselj, besseli, simplify, factor_terms
+    from sympy import sinh, cosh, besselj, besseli, simplify, factor_terms,\
+        DiracDelta
     ILT = inverse_laplace_transform
     a, b, c, = symbols('a b c', positive=True)
     t = symbols('t')
 
     def simp_hyp(expr):
         return factor_terms(expand_mul(expr)).rewrite(sin)
 
-    # just test inverses of all of the above
-    assert ILT(1/s, s, t) == Heaviside(t)
+    assert ILT(1 , s, t) == DiracDelta(t)
+    assert ILT(1/s , s, t) == Heaviside(t)
+    assert ILT(a/(a + s) , s, t) == a*exp(-a*t)*Heaviside(t)
+    assert ILT(s/(a + s) , s, t) == -a*exp(-a*t)*Heaviside(t) + DiracDelta(t)
+    assert ILT((a + s)**(-2) , s, t) == t*exp(-a*t)*Heaviside(t)
```

Comment:
The general convention in Python code (as in English) is to have one space after a comma and no space before.

Reply:
Well spotted, thanks. I made the new asserts with a code generator ;))